new-module/liquidations.md

```diff
@@ -0,0 +1,111 @@
+
+**TLDR**
+* Vaults with a health factor below one risk getting liquidated
+* Liquidated vaults can lose part or all their collateral
+* Angle has special liquidations features such as a variable liquidation amount, a dynamic discount for liquidators, and a discount booster for veANGLE holders
+
+## How liquidations work
+
+### Health Factor
+Each vault has its own collateral ratio. Together with the collateral factor, they are used to compute the health factor of an open vault. For example:
+
+$$
+\texttt{HF = CR * CF} \\
+\texttt{HF} = 160/100 * 2/3 = 1.07
+$$
+
+If this health factor goes below 1, it means that the value of the collateral backing the stablecoin is too low compared to what was defined by the protocol and the position can get liquidated. 
+
+### What happens to a position getting liquidated
+
+Liquidating a position means repaying part or the entirety of the vault's debt (agTokens) against the collateral it holds (ETH or any other collateral token), usually at a discount.
+
+After getting liquidated, a vault will either be closed, or both the amount of debt and collateral will have been reduced in a way that puts the Health Factor of the vault back above a heatlhy target. 
+
+This means that users will either lose their collateral completely, but won't have anything to pay back anything to the protocol, or lose part of their collateral and still have some debt towards the protocol. 
+
+## Details about liquidations within Angle 
+
+### Liquidation Surcharge (fee)
+When a vault gets fully or partially liquidated, a liquidation surcharge is applied to what is being paid by the liquidator and captured by the protocol. In practice, this means that if the liquidation surcharge is 2%, when a liquidator repays 100 of the debt of a vault only 98 will be repaid and 2 will go to the protocol. 
+
+This is a way for the protocol to grow reserves especially from collaterals that often get liquidated. 
+
+### Variable Liquidations Amount
+
+In Angle Borrow module, **the amount of debt to repay during a liquidation is dynamic** and computed such that after the liquidation, the liquidated vault ends up in a level of “health” defined by a target parameter: the [target health factor](/new-module/glossary.md). In some conditions, liquidators may be able to liquidate all of a position’s available debt or collateral to avoid leaving an amount of debt that is too small for the owner to be incentivized to come to repay. 
+
+In most cases though, liquidated vaults may get less than 50% of their position liquidated: this system is designed to be far more borrower friendly than existing alternatives.
+
+More details in section 4.3 of the [Angle Borrow Module Whitepaper](/whitepaper.md#angle-borrowing-module).
```

Comment:
Let's try not to link to the whitepaper: this docs should be a one stop place, with no external references

Reply:
I don't think we should add the maths here, it makes no sense to just copy paste all the whitepaper
If some people want all the details they should read the wp